source/blog/2016-04-08-new-in-rdo-repos-one-size-doesn-t-fit-all.html.md

```diff
@@ -0,0 +1,46 @@
+- Packagers need to have full control over the packaging process.
+
+Provide a distribution suitable for all these scenarios is not an easy task and RDO community ended up with two different distribution set of repos targetting different purposes:
+
+### RDO Trunk repos
+Provide RPM package repositories for OpenStack projects as close as possible to the vanilla upstream code in order to test any change and catch any bug or impact on packaging as soon as possible. These packages are built when a commit is merged in upstream master or any of the supported stable branches. Once built is done, a set of CI jobs is passed to validate installation and basic functionalities. As part of RDO Trunk we provide different repos for CentOS 7 under http://buildlogs.centos.org/centos/7/cloud/x86_64/ :
+
+- **rdo-trunk-&lt;branch>:** provides the latest snapshot of packages successfully built (i.e. rdo-trunk-mitaka).
+- **rdo-trunk-&lt;branch>-tested:** provides the latest snapshot of packages built and with CI jobs successfully passed (i.e. rdo-trunk-master-tested).
+- **rdo-trunk-master-tripleo:** provides the latest snapshot of packages tested by specific TripleO CI jobs on master branch.
+
+Currently (apr '16'), &lt;branch> can be master, mitaka, liberty or kilo.
+
+As operators, these repos may not be the best option to run a stable OpenStack cloud but it can be a good choice if you are interested in deploying an environment to test latest changes in OpenStack code. If you are a OpenStack developer it may be a good option to build environments with latest vanilla upstream code.
+
+Finally, if you are a packager, you probably want to know how the build process went for a specific commit or even to perform an installation based on it. RDO provides both build reports and RPM repos based on each commit for both CentOS and Fedora from https://trunk.rdoproject.org.
+
+[Fred](http://blogs.rdoproject.org/author/fred) wrote an excelent [post](http://blogs.rdoproject.org/7834/delorean-openstack-packages-from-the-future) describing how DLRN (f.k.a. delorean) build packages for RDO trunk repos.
+
+### RDO Stable repos
+Provide a more stable set of packages throguh CentOS CloudSIG repos based on [CentOS Community Build System](https://wiki.centos.org/HowTos/CommunityBuildSystem). These repos differs from RDO Trunk in several ways:
+
+- Packages are only created after GA of a upstream release (only stable branches are used, not master).
+- New packages are created only when a new release is published (release tag created) on upstream repositories. Rebuild for those packages is not automatically triggered so it may take some time to deliver them.
+- In addition to the vanilla upstream code, some patches may be backported to fix security issues or critical bugs not backported upstream. Note that an upstream-first policy is applied so these patches will be applied only after merged in upstream master.
```

Comment:
In addition to critical security and show-stopper issues, also allowed are source code patches to help packaging, prime example is oslo.config patch[1] to enabled distro defaults from dist.conf files

[1] https://github.com/openstack-packages/python-oslo-config/blob/rpm-master/0001-add-usr-share-project-dist.conf-to-the-default-confi.patch